slab/filter.py

```diff
@@ -30,6 +31,18 @@ class Filter(Signal):
 	frequencies = property(fget=lambda self: numpy.fft.rfftfreq(self.ntaps*2-1, d=1/self.samplerate) if not self.fir else None, doc='The frequency axis of the filter.') # TODO: freqs for FIR?
 
 	def __init__(self, data, samplerate=None, fir=True):
+
+		if isinstance(data, str): # load data from a file
+			if samplerate is not None:
+				raise ValueError('Cannot specify samplerate when initialising Sound from a file.')
+			if data[-3::]=='wav':
+				_ = sound.Sound.read(data)
```

Comment:
No. If you have saved a filter as a sound file you are asking for trouble. I dont think its a good idea to permit this. The filter class saves filters as npy files, which is good for now.